config/docker-compose.Linux.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.5'
+
+services:
+  elasticsearch:
+    network_mode: host
```

Comment:
Why do we use `network_mode` instead of `networks`? The Docker docs aren't too clear on this...

Reply:
The answer is a little convoluted, but basically because it is fewer lines and simpler to understand.

The docs in question are [here](https://docs.docker.com/compose/compose-file/#host-or-none), they say you can create a network with driver "host", but that will only be respected for docker-swarm, not docker-compose. There is a way to create a network which uses host networking for docker-compose, but it is roundabout and in my mind difficult to understand when reading. 

In the end I decided that just using `network_mode: host` would express the intention more clearly. It's annoying that it's different to how we use networking in macOS, but there we are 🤷‍♂️